packages/salesforcedx-utils-vscode/webpack.config.js

```diff
@@ -64,6 +45,8 @@ module.exports = {
   // excluding dependencies from getting bundled
   externals: {
     '@salesforce/core': 'commonjs @salesforce/core',
+    applicationinsights: 'commonjs applicationinsights',
```

Comment:
These two dependencies cannot be bundled so we're ignoring them and expecting them to be available in `node_modules`